src/components/SearchGridFormOriginal.vue

```diff
@@ -131,6 +131,7 @@ export default {
       e.preventDefault();
       if (this.searchTermsModel) {
         this.$emit('onSearchFormSubmit', { query: { q: this.searchTermsModel }, shouldNavigate: true });
+        document.getElementById('searchInput').blur();
```

Comment:
The use of `document.getElementById` is discouraged on VueJS. Can you use this instead? https://vuejs.org/v2/guide/components-edge-cases.html#Accessing-Child-Component-Instances-amp-Child-Elements

Reply:
Ok, I got it.